/cm/shared/apps/python/3.5.2/lib/python3.5/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
parameters: {'validation_ratio': 0.1, 'joint_training': True, 'num_embedding_features': 100, 'lr_halve_interval': 15000.0, 'model_load_path': 'models/VDCNN/ag_news-2000_model.pt', 'model_folder': 'models/VDCNN/VDCNN_ag_news---ng20_depth@49/1', 'seed': 1337, 'last_pooling_layer': 'k-max-pooling', 'test_dataset': 'ag_news', 'chunk_size': 2048, 'depth': 49, 'gpu': True, 'freeze_pre_trained_layers': False, 'shortcut': False, 'class_weights': None, 'maxlen': 1024, 'dataset': 'ng20', 'test_interval': 1000, 'transfer_lr': 0.001, 'lr': 0.01, 'test_batch_size': 128, 'combined_datasets': 'ag_news---ng20', 'iterations': 50000, 'model_save_path': 'models/VDCNN/VDCNN_ag_news---ng20_depth@49/1', 'joint_ratio': 0.01, 'joint_test': 0, 'transfer_weights': False, 'shuffle': True, 'test_only': 0, 'batch_size': 128, 'num_prev_classes': 20}
Joint Training !
dataset: AgNews, n_classes: 4
  - loading dataset...
dataset: Newsgroup20, n_classes: 20
  - loading dataset...
Both datasets loaded, going to mix ...
Transfer Data Size: 13179
Transfer Ratio Size: 132
Root dataset Test length - sentences 7600, labels 7600
Transfer dataset Test length - sentences 5645, labels 5645
Unused dataset Test length - sentences 13047, labels 13047
Mixed dataset Test length - sentences 26292, labels 26292
Mixed dataset Train length - sentences 120132, labels 120132
 Joint Training: Txt vectorization...
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 282, in joint_train
    logger.info("n_txt_feats before overriding: ", n_txt_feats)
Message: 'n_txt_feats before overriding: '
Arguments: (77,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 282, in joint_train
    logger.info("n_txt_feats before overriding: ", n_txt_feats)
Message: 'n_txt_feats before overriding: '
Arguments: (77,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 284, in joint_train
    logger.info("Number of classes in the mixed dataset are: ", n_classes, type(n_classes))
Message: 'Number of classes in the mixed dataset are: '
Arguments: (24, <class 'int'>)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 284, in joint_train
    logger.info("Number of classes in the mixed dataset are: ", n_classes, type(n_classes))
Message: 'Number of classes in the mixed dataset are: '
Arguments: (24, <class 'int'>)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 287, in joint_train
    logger.info("Seed for random numbers: ", torch.initial_seed())
Message: 'Seed for random numbers: '
Arguments: (1337,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 287, in joint_train
    logger.info("Seed for random numbers: ", torch.initial_seed())
Message: 'Seed for random numbers: '
Arguments: (1337,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 290, in joint_train
    logger.info("Overriding the number of embedding features to: ", n_txt_feats)
Message: 'Overriding the number of embedding features to: '
Arguments: (100,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 290, in joint_train
    logger.info("Overriding the number of embedding features to: ", n_txt_feats)
Message: 'Overriding the number of embedding features to: '
Arguments: (100,)
Joint training
Setting the lr to : 0.01
/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py:148: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  yhat = nn.functional.softmax(yhat)
Mixed_ag_news---ng20_0.01 - Iter [0/50000]: train-metrics- {'accuracy': 0.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.0, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [1000/50000]: train-metrics- {'accuracy': 0.265625, 'logloss': -1} ; val-metrics- {'accuracy': 0.3500374594189628, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [2000/50000]: train-metrics- {'accuracy': 0.515625, 'logloss': -1} ; val-metrics- {'accuracy': 0.4094730708399234, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [3000/50000]: train-metrics- {'accuracy': 0.640625, 'logloss': -1} ; val-metrics- {'accuracy': 0.3305585615583118, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [4000/50000]: train-metrics- {'accuracy': 0.6328125, 'logloss': -1} ; val-metrics- {'accuracy': 0.5565637226338134, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [5000/50000]: train-metrics- {'accuracy': 0.7421875, 'logloss': -1} ; val-metrics- {'accuracy': 0.5548156164155498, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [6000/50000]: train-metrics- {'accuracy': 0.78125, 'logloss': -1} ; val-metrics- {'accuracy': 0.6770165653874969, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [7000/50000]: train-metrics- {'accuracy': 0.7890625, 'logloss': -1} ; val-metrics- {'accuracy': 0.7064846416382252, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [8000/50000]: train-metrics- {'accuracy': 0.8359375, 'logloss': -1} ; val-metrics- {'accuracy': 0.6391409306584533, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [9000/50000]: train-metrics- {'accuracy': 0.8359375, 'logloss': -1} ; val-metrics- {'accuracy': 0.7331224506784317, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [10000/50000]: train-metrics- {'accuracy': 0.8828125, 'logloss': -1} ; val-metrics- {'accuracy': 0.7252975942728711, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [11000/50000]: train-metrics- {'accuracy': 0.84375, 'logloss': -1} ; val-metrics- {'accuracy': 0.5796220760842421, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [12000/50000]: train-metrics- {'accuracy': 0.84375, 'logloss': -1} ; val-metrics- {'accuracy': 0.6560392907683343, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [13000/50000]: train-metrics- {'accuracy': 0.84375, 'logloss': -1} ; val-metrics- {'accuracy': 0.7648380920669275, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [14000/50000]: train-metrics- {'accuracy': 0.875, 'logloss': -1} ; val-metrics- {'accuracy': 0.7577624240406227, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [15000/50000]: train-metrics- {'accuracy': 0.8828125, 'logloss': -1} ; val-metrics- {'accuracy': 0.7791559144260385, 'logloss': -1}
new lr: 0.005
Mixed_ag_news---ng20_0.01 - Iter [16000/50000]: train-metrics- {'accuracy': 0.9296875, 'logloss': -1} ; val-metrics- {'accuracy': 0.7964704903021727, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [17000/50000]: train-metrics- {'accuracy': 0.90625, 'logloss': -1} ; val-metrics- {'accuracy': 0.8246899192541414, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [18000/50000]: train-metrics- {'accuracy': 0.96875, 'logloss': -1} ; val-metrics- {'accuracy': 0.8100391242820278, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [19000/50000]: train-metrics- {'accuracy': 0.875, 'logloss': -1} ; val-metrics- {'accuracy': 0.7894780654291185, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [20000/50000]: train-metrics- {'accuracy': 0.9140625, 'logloss': -1} ; val-metrics- {'accuracy': 0.8103720968950304, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [21000/50000]: train-metrics- {'accuracy': 0.921875, 'logloss': -1} ; val-metrics- {'accuracy': 0.8121202031132939, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [22000/50000]: train-metrics- {'accuracy': 0.890625, 'logloss': -1} ; val-metrics- {'accuracy': 0.7919753600266378, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [23000/50000]: train-metrics- {'accuracy': 0.8984375, 'logloss': -1} ; val-metrics- {'accuracy': 0.8057104803129943, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [24000/50000]: train-metrics- {'accuracy': 0.9296875, 'logloss': -1} ; val-metrics- {'accuracy': 0.7967202197619245, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [25000/50000]: train-metrics- {'accuracy': 0.8984375, 'logloss': -1} ; val-metrics- {'accuracy': 0.8171980354615833, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [26000/50000]: train-metrics- {'accuracy': 0.84375, 'logloss': -1} ; val-metrics- {'accuracy': 0.8090402064430201, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [27000/50000]: train-metrics- {'accuracy': 0.9609375, 'logloss': -1} ; val-metrics- {'accuracy': 0.7969699492216765, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [28000/50000]: train-metrics- {'accuracy': 0.8984375, 'logloss': -1} ; val-metrics- {'accuracy': 0.8154499292433197, 'logloss': -1}
Mixed_ag_news---ng20_0.01 - Iter [29000/50000]: train-metrics- {'accuracy': 0.8671875, 'logloss': -1} ; val-metrics- {'accuracy': 0.8151169566303171, 'logloss': -1}
slurmstepd: error: *** JOB 3506835 ON node004 CANCELLED AT 2018-03-11T00:20:21 DUE TO TIME LIMIT ***
